sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -739,6 +740,10 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
   ): Promise<ServiceBusMessageImpl[]> {
     throwErrorIfConnectionClosed(this._context);
 
+    if (sequenceNumbers.length === 0) {
+      throw new TypeError("At least one sequence number must be specified.");
```

Comment:
Interesting... Elsewhere in #12207, I am going with returning an empty array...

Reply:
Other languages are throwing.

Reply:
That feels excessive... Say I am getting the input array from another part of my code which I simply pass to `receiveDeferredMessages()`. If the client throws, then the customer always has to do a length check on the input. If we do a no-op, then things are gracefully handled.

Reply:
Check in with the feature crew and make sure this is okay - I was merely aping what already existed :)